app/src/main/java/de/westnordost/streetcomplete/util/location/FineLocationManager.kt

```diff
@@ -86,7 +86,7 @@ class FineLocationManager(context: Context, locationUpdateCallback: (Location) -
 
 // Based on https://web.archive.org/web/20180424190538/https://developer.android.com/guide/topics/location/strategies.html#BestEstimate
 
-private const val TWO_MINUTES = 1000L * 60 * 2
+private const val TWO_MINUTES = 1000L * 1000 * 1000 * 60 * 2
```

Comment:
uh what? This ain't two minutes :-D

Reply:
It looks like a lot, but actually it is...

Before Android 13 location can only return `elapsedRealtimeNanos`, but not `elapsedRealtime` (milliseconds), so I needed to convert the 2 minutes from milliseconds to nanoseconds.
I'll rename it to something like `TWO_MINUTES_IN_NANOSECONDS`.